core/domain/algorithm_proto_registry.py

```diff
@@ -0,0 +1,83 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Registry for proto model corresponding to ML classifiers."""
+
+from __future__ import absolute_import # pylint: disable=import-only-modules
+from __future__ import unicode_literals # pylint: disable=import-only-modules
+
+from core.domain.proto import text_classifier_pb2
+
+import python_utils
+
+
+class Registry(python_utils.OBJECT):
+    """Registry of all classifier classes."""
+
+    CLASSIFIER_TO_PROTO_MAPPING = {
```

Comment:
Perhaps "proto class"? "Proto" suggests "proto instance", by default.

Reply:
This registry is no longer needed. Removed it.